Dynatrace/dynatrace-operator
# Fix e2e framework not stopping if setup fails

TIcket: https://dt-rnd.atlassian.net/browse/K8S-11312


## Description

In the latests version of the e2e framework (0.4.0 ) has a bug in it that when a setup step fails it will only skip the other setup steps and not the whole test.

Which is unexpected behaviour for us, as logically, why would I want to run tests against an incorrectly setup env?

## How can this be tested?

Check e2e and check if the setup fails by having a wrong CRD e.g tests should stop
